docs/readthedocs/source/doc/LLM/Quickstart/fastchat_quickstart.md

```diff
@@ -0,0 +1,96 @@
+# Serving using IPEX-LLM and FastChat
+
+FastChat is an open platform for training, serving, and evaluating large language model based chatbots. You can find the detailed information at their [homepage](https://github.com/lm-sys/FastChat).
+
+IPEX-LLM can be easily integrated into FastChat so that user can use `IPEX-LLM` as a serving backend in the deployment.
+
+## Quick Start
+This quickstart guide walks you through installing and running `FastChat` with `ipex-llm`.
+
+## 1. Install IPEX-LLM with FastChat
+
+```bash
+pip install --pre --upgrade ipex-llm[serving]
+
+# Or
+pip install --pre --upgrade ipex-llm[all]
+```
+
+To add GPU support for FastChat, you may install **`ipex-llm`** as follows:
+
+```bash
+pip install --pre --upgrade ipex-llm[xpu,serving] --extra-index-url https://pytorch-extension.intel.com/release-whl/stable/xpu/us/
+
+```
+
+## 2. Start the service
+
+### Launch controller
+
+You need first run the fastchat controller
+
+```bash
+python3 -m fastchat.serve.controller
+```
+
+### Launch model worker(s) and load models
+
+Using IPEX-LLM in FastChat does not impose any new limitations on model usage. Therefore, all Hugging Face Transformer models can be utilized in FastChat.
+
+#### IPEX-LLM worker
+
+To integrate IPEX-LLM with `FastChat` efficiently, we have provided a new model_worker implementation named `ipex_llm_worker.py`.
+
+```eval_rst
```

Comment:
I am not sure if this is the correct format or I am doing something wrong..
![image](https://github.com/intel-analytics/ipex-llm/assets/110874468/35949e6b-3cf8-4a30-8a13-3862d7c659f8)

Reply:
It will display when convert to html，but I think it's not necessary to use this. I will adjust above issues.